cache.go

```diff
@@ -183,17 +185,24 @@ func (c *Cache) Get(key interface{}) (interface{}, bool) {
 // To dynamically evaluate the items cost using the Config.Coster function, set
 // the cost parameter to 0 and Coster will be ran when needed in order to find
 // the items true cost.
-func (c *Cache) Set(key, value interface{}, cost int64) bool {
+//
+// TTL is the amount of time (in seconds) that the item will remain in the
+// cache. To make an item stay indefinitely, set TTL to -1.
+func (c *Cache) Set(key, value interface{}, cost, ttl int64) bool {
```

Comment:
This would be the moment to start tagging releases. Because you haven't tagged a release before everyone is using `master` and everyone's build will break at soon as this is merged.

Reply:
Agreed